include/z64.h

```diff
@@ -739,7 +721,7 @@ typedef struct {
     /* 0x80 */ OSMesg unk_80;
     /* 0x84 */ f32 unk_84;
     /* 0x88 */ f32 unk_88;
-    /* 0x8C */ LightSettings2 unk_8C;
+    /* 0x8C */ LightSettingsAdjustments lightAdjustments;
```

Comment:
```suggestion
    /* 0x8C */ EnvLightSettings lightSettings;
```
This is what this is called in OoT. Is it different enough in MM to have a different name?

Reply:
I don't think so. I probably grabbed an old name instead of the current one :sweat_smile: